cypress/e2e/signUp.cy.js

```diff
@@ -1,5 +1,29 @@
+/// <reference types="cypress" />
+const { generateUser } = require('../support/generate');
+
 describe('Sign Up page', () => {
+  before(() => {cy.visit('https://react-redux.realworld.io');});
   it('should provide an ability to register new account', () => {
-
+    const user = generateUser();
+    
+  
+    cy.get(':nth-child(3) > .nav-link')
+    .click ();
+    cy.contains('h1', 'Sign Up')
+    .should('be.visible');
+    cy.url()
+    .should('include', 'register');
```

Comment:
```suggestion
    cy.get(':nth-child(3) > .nav-link')
      .click ();
    cy.contains('h1', 'Sign Up')
      .should('be.visible');
    cy.url()
      .should('include', 'register');
```
please format the code